### DISCUSSION TITLE
trusted_proxies not set

### DISCUSSION THREAD

**Discussion by wuast94** [nextcloud/all-in-one#5660] [Questions] [answered]
i get the error that my trusted_proxies are not set correctly.

```
'trusted_proxies' => 
  array (
    0 => '127.0.0.1',
    1 => '::1',
    10 => '',
  ),
  ```
  
  and indeed they arent, i followed the manual install and run thin in a docker swarm cluster.

**Comment by szaimen** [comment]
Hi, this is a bug currently and will be fixed with https://github.com/nextcloud/all-in-one/pull/5666

**Reply by wuast94** [reply]
im pretty sure that this will not fix the problem, i am using the nextcloud-aio network, and the pr didnt change more than adding this back. 

BUT it could be becouse i am using a second network for my traefik proxy attached to the apache container with
```
networks:
      - nextcloud-aio
      - proxy_t2_proxy
```

**Reply by wuast94** [reply]
also my nextcloud container is in the nextcloud-aio network and in another one i need for some special access.
also becouse it wasnt that clear, all my nextcloud containers are in the nextcloud-aio network

**Comment by szaimen** [comment]
Hey, thanks for coming back to me. You are right. This is not the correct fix.
See https://github.com/nextcloud/all-in-one/blob/6e2a14f1c7ef936a06612dbfb747b08dd6505724/Containers/nextcloud/entrypoint.sh#L596-L600

**Reply by wuast94** [reply]
i think the dig itself in a swarm cluster isnt the best way anyway, espacially with multiple networks attached to it i guess.

maybee we could add a ENV to "hardcode" the IPv4_ADDRESS variable? but this also dont sound like the right way. the only ip adress that matters here is the one of the reverse proxy, but this one cant nextcloud know becouse the apache container is in beetween and "isolates" the two.

wgat do you think about this @szaimen ? catch me if im missing anything or didnt understand right :)

 i would be happy to make a PR for this.

**Comment by szaimen** [comment, marked as answer]
This will be adjusted with https://github.com/nextcloud/all-in-one/pull/5671
